libs/design-system/input/src/lib/input-styles/input-styles.component.scss

```diff
@@ -0,0 +1,35 @@
+.mat-mdc-form-field {
+  font-family: 'Metropolis';
```

Comment:
Read from a --sys variable. Add `-font` to get just the font family from the variables, i.e. `--sys-label-small-font`